pkg/app/piped/cloudprovider/kubernetes/diff.go

```diff
@@ -46,6 +50,13 @@ type DiffListChange struct {
 }
 
 func Diff(old, new Manifest, opts ...diff.Option) (*diff.Result, error) {
+	if old.Key.IsSecret() && new.Key.IsSecret() {
+		var err error
+		new.u, err = normalizeNewSecret(old.u, new.u)
+		if err != nil {
+			return nil, err
+		}
```

Comment:
```suggestion
		if new.u, err := normalizeNewSecret(old.u, new.u); err != nil {
			return nil, err
		}
```
nits

Reply:
I think `new.u` cannot come left side of `:=`.

Reply:
ah, got it, thank you 👍